build.sh

```diff
@@ -174,35 +236,19 @@ else
     KSP_NAME=$ghprbActualCommit
 fi
 
-mono --debug ckan.exe ksp add ${KSP_NAME} "`pwd`/dummy_ksp"
-mono --debug ckan.exe ksp default ${KSP_NAME}
-
-echo Running ckan update
-mono --debug ckan.exe update
-
```

Comment:
We're doing these per file as it is, I've seen it barf when we've tried to do it out here.